### ISSUE TITLE
Clarify how to set up performance monitoring

### ISSUE THREAD

**Issue by warpr** [getsentry/sentry-docs#4154]
### Core or SDK?

Platform/SDK

### Which part? Which one?

Performance Monitoring

### Description

I'm trying to setup performance monitoring, but the documentation on how to do this is unclear, the first sentence is:

> If you don't already have performance monitoring enabled, use the link below to quickly set up access to our Performance Monitoring features. 

"The link below" in this sentence is unclear -- there are many links on this page, and none of them seem to enable performance monitoring.

It also mentions this:
> Customers on legacy plans must add transaction events to their subscription in order to access our performance monitoring features.

But gives no guidance on how you would know whether you're on a legacy plan or not.  Presumably I'm not, as I see no indication of it on my billing screen.

### Suggested Solution

- Clarify how to enable performance monitoring when not on a legacy plan.
- Clarify how as a team member I can determine whether our account is on a legacy plan or not.

**Comment by imatwawana** [comment]
@warpr Thanks for flagging this! The wording of the first sentence does need to be updated as it's meant to point you to the list of supported SDKs below where you can click the appropriate link. Each of those links takes you to the directions for setting up performance for the SDK. 

As for the legacy plans issue, I'm tagging the team I think can shed some light here.

**Comment by getsentry-release** [comment]
Routing to @getsentry/revenue for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

**Comment by markstory** [comment]
:wave: If you see a block like 

![Screen Shot 2021-09-15 at 4 57 20 PM](https://user-images.githubusercontent.com/24086/133508555-21f9b5a4-f8d1-4e1f-9131-65f024fe7c31.png)

on your organization's billing overview (the view with your usage chart) you're on a legacy plan that doesn't support transactions. Unless you specifically click on 'Show legacy plans' during checkout you'll only be shown plans that *do* have performance monitoring and transactions. You should also see 'Transactions' in step 2 of the checkout process where you choose your reserved limits.

If that is helpful context I can try to clarify that section of the docs.

**Comment by warpr** [comment]
@markstory The issue is now resolved for me -- I was trying to enable performance monitoring in two accounts:

1. My employers account, where I am a "Manager", so I do not have access to the billing screen.
2. My personal free trial account, which presumably didn't have transaction support because it's a free trial.

In both cases clicking the "Performance" option in the left sidebar menu took me to a blank slate which didn't give any information about why performance monitoring wasn't enabled yet, even though I was sending tracing events to the account.

In both accounts it is enabled now, in the free account because I clicked "Try Business", which presumably temporarily added transaction support to my account.  In my employers account it is still a mystery to me why it suddenly enabled on its own, hours after sending the first tracing events.
